resources/views/checkout/partials/progressbar.blade.php

```diff
@@ -1,23 +1,20 @@
-{{-- TODO: When the steps count change this grids doesn't play nice --}}
-<nav class="grid grid-cols-12 my-5">
+
+<nav class="flex my-5">
     @foreach ($checkoutSteps as $checkoutStepKey => $checkoutStep)
-        <a href="{{ route('checkout', $checkoutStep) }}" @class([
-            'text-center col-span-3 relative focus:outline-none',
-            'pointer-events-none cursor-default' => $currentStepKey < $checkoutStepKey,
-        ])>
+        <div class="text-center w-full relative focus:outline-none">
             @if (!$loop->last)
-                {{-- TODO: This line is clickable which it shouldn't --}}
                 <div class="absolute flex w-full h-0.5 top-5 left-1/2 {{ $currentStepKey > $checkoutStepKey ? 'bg-neutral' : 'bg-inactive' }}"></div>
             @endif
-            <div @class([
+            <a href="{{ route('checkout', $checkoutStep) }}" @class([
```

Comment:
Do we want to add a check to prevent clicking ahead, or let the response to wanting to place your order tell you?

Reply:
`pointer-events-none` 🥳 or do you have a cleaner solution?